zkevm-circuits/src/evm_circuit/execution/calldatacopy.rs

```diff
@@ -488,4 +495,61 @@ mod test {
             Word::from(0),
         );
     }
+
+    #[test]
+    fn calldatacopy_gadget_busmapping_internal() {
```

Comment:
you can remove the old testing that manually creates the steps and rw operations.

Reply:
Sure. I also need to do multiple tests with different arguments. Will push in a while.